### PR TITLE
update libqb

### PR DESCRIPTION
**Pull request by chifu1234**

### CODE REVIEW

File: srcpkgs/libqb/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'libqb'
 pkgname=libqb
-version=1.0.5
+version=2.0.0
 revision=1
 build_style=gnu-configure
+hostmakedepends="pkg-config libxml++-devel"
 short_desc="High performance logging, tracing, ipc, poll"
 maintainer="Kevin Klopfenstein <kk@sudo-i.net>"
```
Comment (sgn) [inline srcpkgs/libqb/template:8]: This is your name, but you're also listed as first time contributor?

**Comment by sgn** [comment]
- Please use formated commit message.
- Please fix cross compile.

**Comment by Chocimier** [comment]
Moving libxml++-devel to makedepends should fix cross builds.

As soname changed, common/shlibs should be updated, and all packages depending on libqb should be revbumped.

Please leave empty line after variables.

File: srcpkgs/libqb/template (hunk 1 above)
Comment (Chocimier) [reply srcpkgs/libqb/template:8]: Nothing wrong if address is real, it just means address is not attached to gh account.

**Comment by sgn** [comment]
b12c47697ea035ff8eb0f86f1f8afbb62133b9d5

**Pull request closed without merging**